src/OrchardCore/OrchardCore.ContentLocalization.Abstractions/Records/Migrations.cs

```diff
@@ -25,17 +25,24 @@ public int Create()
                 .Column<string>("LocalizationSet", col => col.WithLength(26))
                 .Column<string>("Culture", col => col.WithLength(16))
                 .Column<string>("ContentItemId", c => c.WithLength(26))
+                .Column<bool>("Published")
+                .Column<bool>("Latest")
             );
 
             SchemaBuilder.AlterIndexTable<LocalizedContentItemIndex>(table => table
-                .CreateIndex("IDX_LocalizationPartIndex_LocalizationSet_Culture", new[] { "LocalizationSet", "Culture" })
+                .CreateIndex("IDX_LocalizationPartIndex_LocalizationSet_Culture", "LocalizationSet", "Culture")
```

Comment:
JT should we just drop this one (from the code, not from existing sites). It looks useless now?

Reply:
Okay